README.NewResourceFromTerraform.md

```diff
@@ -180,6 +180,8 @@ ServiceMappings file. Add the `ResourceConfig` for your resource:
     1.  Set `v1alpha1ToV1beta1` to `true`.
     1.  Set `storageVersion` to `v1alpha1`.
 
+    > NOTE: For [autogenereated resources](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/2f4f2c6ee65034f7bbb4b4f62d2ba15842ff114d/scripts/resource-autogen/generated/servicemappings), don't remove the service mappings. Instead copy the file over to `config/servicemappings` .
```

Comment:
Could you update the last sentence to something like:

```
Instead copy the content or the file over to [config/servicemappings/](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/v1.119.0/config/servicemappings).
```

Highlighting the path of `config/servicemappings` may confuse first time contributor because they may not be familiar with this folder. Also the space between \` and \. made \. the first and only character in a new line: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/0053abf1a5631d55024e2cdf7a9d19523703cfa9/README.NewResourceFromTerraform.md#add-the-resource-config-to-service-mappings